.travis.yml

```diff
@@ -121,7 +121,8 @@ script:
         CGO_ENABLED=0 go test -c -o deviceadm -coverpkg $(go list ./... | grep -v vendor | grep -v mocks | grep -v test | tr  '\n' ,);
         sudo docker build -f Dockerfile.acceptance-testing -t mendersoftware/deviceadm:prtest .;
         ./tests/build-acceptance ./tests ./docs/management_api.yml ./docs/internal_api.yml;
-        TESTS_DIR=$PWD/tests ./tests/run-test-environment "acceptance" $PWD/integration_new ./tests/docker-compose.yml;
+        TESTS_DIR=$PWD/tests ./tests/run-test-environment "acceptance" $PWD/integration_new ./tests/docker-compose-acceptance.yml;
```

Comment:
use && to start tenant tests after non-tenant tests, otherwise failure in non-tenant tests will not fail the build step